src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { FileHandle } from '@eco-news-models/create-news-interface';
+import { ImageCroppedEvent } from 'ngx-image-cropper';
+import { EditProfileService } from '@global-user/services/edit-profile.service';
+import { MatDialog } from '@angular/material';
+import { ErrorComponent } from '@errors/error/error.component';
```

Comment:
Please, sort imports in such order: angular files, core files/libraries (RxJS etc.), local files

Reply:
fixed